tests/publiccloud/az_accelerated_net.pm

```diff
@@ -73,6 +73,7 @@ sub enable_accelerated_net {
     sleep 60 * 3;    # Sometimes, IP is not reachable after the restart and 5 minutes is enough.
     $instance->{private_ip} = get_new_ip($instance);
     $instance->public_ip(get_new_ip($instance, 1));
+    script_run("ssh-keyscan " . $instance->public_ip . " | tee ~/.ssh/known_hosts /home/$testapi::username/.ssh/known_hosts");
```

Comment:
we doing the same in the `$instance->wait_for_ssh`

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/publiccloud/instance.pm#L460 why this is not enough ? or it is skipped somehow in this case ?

Reply:
In this case the IP address is changed after we modify the network. Plus in this case we have 2 VMs instead of 1.

Reply:
ah ok make sense